packages/dotcom-ui-base-styles/src/lib/fontFaces.ts

```diff
@@ -1,8 +1,8 @@
 // NOTE: please also update ./styles.css when updating the URLs defined below. They
 // need to stay in-sync to ensure our CSS output matches the resource hints we set.
 export const fontFaceURLs = [
```

Comment:
Should we see both woff and woff2 files in this array now, rather than just woff2?

Reply:
No because this is used for preloading, and we don't want both formats to preload. I think we're safe to just preload woff2 as it looks like browsers which support preloading also support woff2 :)

Reply:
Nice one 👍